backend/danswer/llm/answering/answer.py

```diff
@@ -112,6 +115,7 @@ def __init__(
         skip_explicit_tool_calling: bool = False,
         # Returns the full document sections text from the search tool
         return_contexts: bool = False,
+        skip_generation: bool = False,
```

Comment:
can be even more explicit here with the naming since generation could mean a few things, like we're using a generator for example. Maybe call it "skip_gen_ai_answer_generation"?